osmnx/elevation.py

```diff
@@ -6,6 +6,7 @@
 # Web: https://github.com/gboeing/osmnx
 ################################################################################
 
+import demquery
```

Comment:
This should be handled as an optional import. For example see how scikit-learn components are imported here: https://github.com/gboeing/osmnx/blob/bde270d4a148d8f8911aef59c86a099adc76c24f/osmnx/geo_utils.py#L39-L42
And then checked/handled here: https://github.com/gboeing/osmnx/blob/bde270d4a148d8f8911aef59c86a099adc76c24f/osmnx/geo_utils.py#L427-L428

Reply:
Ah makes sense. I wasn't sure how to do optional imports but I agree it should be one